mongodbatlas/data_source_mongodbatlas_third_party_integration_test.go

```diff
@@ -243,22 +212,14 @@ func testAccMongoDBAtlasThirdPartyIntegrationResourceConfig(config *thirdPartyCo
 func testAccCreateThirdPartyIntegrationConfig() *matlas.ThirdPartyIntegration {
 	account := testGenString(6, numeric)
 	return &matlas.ThirdPartyIntegration{
-		Type: "OPS_GENIE",
-		// Pager dutty 20-character strings
-		LicenseKey:  testGenString(20, alphabet),
-		APIToken:    fmt.Sprintf("xoxb-%s-%s-%s", testGenString(12, numeric), testGenString(12, numeric), testGenString(24, alphaNum)),
```

Comment:
Same for OPS_GENIE, I see it in the API Doc

Reply:
OPS_GENIE is not being removed the line changed because of formatting. Here I only removed unnecessary attributes from Flowdock & New Relic integration